lib/driver.js

```diff
@@ -116,6 +116,7 @@ class YouiEngineDriver extends BaseDriver {
             await this.device.startSession(caps, this);
             break;
           case "noproxy":
```

Comment:
might need a break for this case

Reply:
no, they are both doing the same action, which is break...